src/quo2/components/inputs/profile_input/style.cljs

```diff
@@ -1,13 +1,12 @@
 (ns quo2.components.inputs.profile-input.style
-  (:require [react-native.platform :as platform]
-            [quo2.foundations.colors :as colors]))
+  (:require [quo2.foundations.colors :as colors]))
 
 (defn container
   [customization-color]
   {:background-color   (colors/custom-color customization-color 50 40)
    :padding-horizontal 12
    :padding-top        12
-   :padding-bottom     (if platform/ios? 10 0)
```

Comment:
Hello @alwx, I've removed this condition in order to resolve the Android issue. Following this change, I couldn't detect any issues, but please let me know if I've overlooked something.

Reply:
Does it still match the design?

Reply:
Yes.